site/src/_layouts/default.hbs

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>d3fc {{#if title}}:{{/if}} {{title}}</title>
+    <meta name="description" content="A collection of components that make it easy to build interactive charts with D3.">
+
+    <script src="{{assets}}/scripts.js"></script>
+
+    <link rel="stylesheet" href="{{assets}}/styles.css" />
+  </head>
```

Comment:
If their browser doesn't support SVG, I'm not sure why they'd be looking at an SVG charting library?
I've added a task to make sure that IE9 renders (it looks like most stuff is working, just edge cases).